JavaCollection/ListExample.java

```diff
@@ -0,0 +1,50 @@
+package JavaCollection;
+
+import java.util.*;
+
+public class ListExample {
+    public void createList() {
+        List<String> exampleList = new ArrayList<>();
+        exampleList.add("One");
+        exampleList.add("Two");
+        exampleList.add("Three");
+        exampleList.add("Four");
+        exampleList.add("Five");
+        exampleList.add("Six");
+        exampleList.add("PleasePrintMe");
+        exampleList.add("DoNotPrint");
+
+        Iterator<String> iterator = exampleList.iterator();
+
+        System.out.print("Joined List Elements are: ");
+        while (iterator.hasNext()) {
+            System.out.print(iterator.next());
+        }
+
+        System.out.print("\nReversed List Elements:");
+        Collections.reverse(exampleList);
+        for (String listElements : exampleList) {
+            System.out.print(" " + listElements);
+        }
+
+        Collections.reverse(exampleList);
+        System.out.println("\nElement with Index 2 is: " + exampleList.get(2));
```

Comment:
In this point it's required to find position of element "Two"
Expected: Element "Two" on the position 1 (just as example)
Actual: Element with Index 2 is: Three